### PR TITLE
Free Pill UI Banner

### LINKED ISSUE #1535: Free Pill
As a free user
I want to see a pill in the top right that says free
So that when I click it takes me to the subscription page

The design for this will follow the same as the Premium pill that appears.

Should direct to:
https://users.nexusmods.com/account/billing/premium?pk_campaign=NexusModsApp

### PR DESCRIPTION
**Pull request by MistaOmega**
Closes #1535 

Premium still working:
image

Free:
image

### CODE REVIEW

**Review by Sewer56 — APPROVED** [review]

**Comment by Pickysaurus** [comment]
We'll need to check with @captainsandypants but I think the "FREE" text not being there is intended.

**Comment by MistaOmega** [comment]
Also looked back at the issue and realised I didn't make it link to the page requested, so I should do that if this ends up still be required

**Comment by captainsandypants** [comment]
I don't mind this being a thing for now but plan to explore having a "Get premium" type button up there

**Comment by Pickysaurus** [comment]
> Also looked back at the issue and realised I didn't make it link to the page requested, so I should do that if this ends up still be required

@MistaOmega if possible can you add the link as the original ticket mentions? I think we can merge it for now. Although as captainsandypants said it will be changed at some point in the future!

File: src/NexusMods.App.UI/Controls/TopBar/TopBarViewModel.cs [hunk 1]
```diff
@@ -139,6 +140,12 @@ public TopBarViewModel(
             await osInterop.OpenUrl(uri);
         });
 
+        OpenNexusModsPremiumCommand = ReactiveCommand.CreateFromTask(async () =>
+        {
+            var uri = new Uri("https://users.nexusmods.com/account/billing/premium?pk_campaign=NexusModsApp");
```
Comment (erri120) [inline src/NexusMods.App.UI/Controls/TopBar/TopBarViewModel.cs:145]: Use the `NexusModsUrlBuilder`:

```csharp
NexusModsUrlBuilder.CreateGenericUri("https://users.nexusmods.com/account/billing/premium");
```

**Review by erri120 — CHANGES_REQUESTED** [review]

**Comment by MistaOmega** [comment]
@erri120 for note i've changed the other 2 uses of new Uri, to use the builder also

**Review by erri120 — APPROVED** [review]

**Pull request merged**